pyls/workspace.py

```diff
@@ -25,7 +25,7 @@ class Workspace(object):
 
     def __init__(self, root, lang_server=None):
         self._url_parsed = urlparse(root)
-        self.root = self._url_parsed.path
+        self.root = unquote(self._url_parsed.path)
```

Comment:
Would it be better to unquote, and then url parse? That way any future references to `self._url_parsed` are correct.